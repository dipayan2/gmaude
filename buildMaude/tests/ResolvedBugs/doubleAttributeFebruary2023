#!/bin/sh

MAUDE_LIB=$srcdir/../../src/Main
export MAUDE_LIB

../../src/Main/maude \
  < $srcdir/doubleAttributeFebruary2023.maude -no-banner \
  > doubleAttributeFebruary2023.out 2>&1

diff $srcdir/doubleAttributeFebruary2023.expected doubleAttributeFebruary2023.out > /dev/null 2>&1
