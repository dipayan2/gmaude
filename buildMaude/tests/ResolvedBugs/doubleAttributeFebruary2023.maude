***
***	Test for warning with double iter attribute.
***

fmod FOO is
  sort Foo .
  op f : Foo -> Foo [iter iter] .
endfm
