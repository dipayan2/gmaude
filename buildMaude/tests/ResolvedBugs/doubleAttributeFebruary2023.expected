Warning: <standard input>, line 7 (fmod FOO): duplicate attributes.
Bye.
